airflow/providers/airbyte/operators/airbyte.py

```diff
@@ -66,6 +67,7 @@ def __init__(
         api_type: Literal["config", "cloud"] = "config",
         wait_seconds: float = 3,
         timeout: float = 3600,
+        namespace: str = "default",
```

Comment:
If we are adding some attribute to the whole Operator and it will be used only by OpenLineage i'd expect the docstring to have some detailed description on why it's needed, how it will be used and why it can't be "extracted" from what's already passed to the operator.